testing/presubmit.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
You could probably share some pieces of this file with continuous.sh. Or use the same script and condition slightly depending on whether KOKORO_GITHUB_PULL_REQUEST_NUMBER is set?

Reply:
Done.